### PR TITLE
Fix invalid package.json; remove "install" run script

### PR DESCRIPTION
**Pull request by junglebarry**
Fixes the following issues

* `package.json` is not valid JSON due to a trailing comma
* `install` run script interferes with the installation as a dependency through `npm`, so I've removed it.

I've bumped the version to `.20` accordingly.

### CODE REVIEW

File: dist/app.min.css [hunk 1]
```diff
@@ -453,7 +453,7 @@ mark,
   text-transform: capitalize; }
 
 .text-muted {
-  color: #8c8c8c; }
+  color: #868e96; }
```
Comment (junglebarry) [inline dist/app.min.css]: I cannot explain why these have been regenerated.  I've not changed anything, just run `npm install`.

File: package.json [hunk 2]
```diff
@@ -37,8 +37,8 @@
   },
   "scripts": {
     "build": "./node_modules/node-sass/bin/node-sass app.scss dist/app.min.css --style compressed",
-    "install": "npm run build",
+    "postinstall": "npm run build",
     "serve": "http-server",
-    "watch": "./node_modules/node-sass/bin/node-sass --watch app.scss dist/app.min.css --style compressed",
+    "watch": "./node_modules/node-sass/bin/node-sass --watch app.scss dist/app.min.css --style compressed"
```
Comment (junglebarry) [inline package.json]: Trailing comma is invalid JSON.

File: package.json [hunk 3]
```diff
@@ -37,8 +37,8 @@
   },
   "scripts": {
     "build": "./node_modules/node-sass/bin/node-sass app.scss dist/app.min.css --style compressed",
-    "install": "npm run build",
+    "postinstall": "npm run build",
```
Comment (junglebarry) [inline package.json]: It seems somehow strange to replace `install`, as this suggests we don't mean to download dependencies - would `postinstall` be more appropriate?

Comment (junglebarry) [reply package.json]: I'm not sure `postinstall` is a good idea either.  The `install` hook is run when you install the project for local dev, but also when installing the project as a dependency.  This just breaks for me when used in Shipshape.

What's the rationale behind adding the hook?

**Comment by pads** [comment]
Bump the version to `.20`

**Pull request merged**